Misc/NEWS.d/next/C API/2022-10-24-12-09-17.gh-issue-98610.PLX2Np.rst

```diff
@@ -0,0 +1,5 @@
+Disallowing subprocess in subinterpreters is no longer supported. Instead,
```

Comment:
double negative... how about wording this like

```
The use of the :mod:`subprocess` is now allowed from subinterpreters. Instead
:func:`os.exec` can now be disallowed.  ... existing text ...
```

Reply:
fixed